exotations/solvers/exotica_ilqr_solver/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2018, University of Edinburgh
```

Comment:
Why do we need a separate license file?

Reply:
Well, AICO and IK have a separate one. The rest don't. What's the rule?